lib/storage/platforms/index.ts

```diff
@@ -0,0 +1,3 @@
+import WebStorage from './WebStorage';
```

Comment:
Can you remind me what the purpose is of this file? I'm wondering why it can't import `IDBKeyValProvider` directly.

Reply:
@tgolen you are right, seems like there is no sense to re-export these files - I removed these index files 👍

Reply:
Ah, sorry, I actually like the index files (because that is a better system for determining which platform uses which provider). Could you remove the other storage files instead?

Reply:
@tgolen sorry, I didn't describe correctly what I did 🤦 

I left `index` and `index.native` files and removed `WebStorage`/`NativeStorage` - this is a commit for a reference https://github.com/Expensify/react-native-onyx/pull/475/commits/53bd1e7173696e3f5a5fb6358923a089194468bf